Package.swift

```diff
@@ -18,5 +18,11 @@ let package = Package(
         .testTarget(
             name: "CrossroadTests",
             dependencies: ["Crossroad"]),
+        .target(
+            name: "Crossroad-tvOS",
```

Comment:
I think you need not add targets for each platforms on `Package.swift`.

The required changes are only L8.

Reply:
You are correct, I have dropped the other changes in this file.